examples/manage_asset_request.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+# This file is part of the Ingram Micro Cloud Blue Connect SDK.
+# Copyright (c) 2019-2020 Ingram Micro. All Rights Reserved.
+import json
+from connect.config import Config
+from connect.resources.fulfillment import FulfillmentResource
+
+
+class AssetRequest():
+    configuration = Config(file='examples/config.json')
+
+    def list_asset_request(self):
+        asset_request = FulfillmentResource(config=self.configuration)
+        return asset_request._asset_requests.search()
```

Comment:
The `FulfillmentResource` should expose a "search_asset_requests" method instead of using the internal `_asset_requests` member.

Reply:
Fixed, thanks!